lib/lotus/generators/application/container/Gemfile.tt

```diff
@@ -16,6 +16,16 @@ gem 'lotusrb',      '<%= Lotus::VERSION %>'
 gem 'lotus-model', '<%= config[:lotus_model_version] %>'
 <%- end -%>
 
+<%- if config[:adapter] == 'mysql' %>
```

Comment:
@Linuus Can you please resolve this in `container.rb`? The pass it to `opts` as `database_gem` and keep the template clean with `gem <%= database_gem %>`.